src/pages/workspaces/workspace/Data.js

```diff
@@ -573,7 +573,7 @@ const WorkspaceData = _.flow(
               onClick: () => setImportingReference(true),
               'aria-haspopup': 'dialog'
             }, [icon('plus-circle', { size: 21 })])
-          }, _.map(type => h(DataTypeButton, {
+          }, [_.map(type => h(DataTypeButton, {
```

Comment:
every other instance of `DataTypeSection` puts their children inside a `[...]`, so I'm doing the same here. I'll also note that every other instance of `DataTypeSection` is mapping over an array of objects, not an array of strings.